cloudprem/static/userdata.yml

```diff
@@ -36,7 +36,15 @@ write_files:
 
     aws eks update-kubeconfig --name "${eks_cluster_name}" --role-arn "${eks_cluster_access_role}" --kubeconfig /home/ssm-user/.kube/config --region ${aws_region}
     chown ssm-user:ssm-user /home/ssm-user/.kube -R
+- path: /tmp/create_frontegg_schema.sh
+  permissions: '700'
+  content: |
+    #!/bin/bash -xe
+    set -o xtrace
+
+    mysql --defaults-file=/home/ssm-user/.my.cnf -e "create database frontegg_events; create database frontegg_webhooks"
```

Comment:
Ehw. We're doing the `frontegg_events` DB creation during bastion boot? That's a little dirty. But, hey, if it works...

Reply:
It's super dirty and I hate it. But short of modifying the frontegg helm charts to add the support for init containers this was the best way I could think of doing it that would work both on a local laptop as well as in the pipeline (terraform has a mysql resource that could do it but that would not work on a laptop due to inaccessibility of the RDS instance from outside the VPC)

Reply:
Oh. Ehw. The frontegg chart doesn't allow init containers. That explains it.